node/test/Infrastructure/Generator.hs

```diff
@@ -129,18 +126,19 @@ simpleModel = GeneratorModel {
       gmAllAddresses  = addrs
     , gmEstimateFee   = \_ _ -> 0
     , gmBootstrapStakeholders = Addr <$> [0..3]
-    , gmBoot          = DSL.Transaction {
+    , gmBoot          = Transaction {
                             trFresh = fromIntegral (length addrs) * initBal
-                          , trIns   = Set.empty
-                          , trOuts  = [DSL.Output a initBal | a <- addrs]
+                          , trIns   = error "Boot inputs"
+                          , trOuts  = (\a -> Output a initBal (balanceStake (a :| []))) <$> addrs
```

Comment:
So I guess the idea here is that all the outputs will assign a stake of 1 to all the addresses. This means that I didn't get how are we encoding a `Repartition`. I thought a repartition assigned portions of the stake to addresses, so that meant that the repartition in the output was a function of the output value. So if we want to translate an output in the DSL of the form:

```haskell
DSL.Output a0 88
```

to an `Abstract.Output`, we could have things like:

```haskell
Abstract.Output a0 88 [(a1, 22, a2 1)]
```

(where `a1`, `a2` are addresses of stakeholders).